apps/i18n/common/en_us.json

```diff
@@ -618,8 +618,8 @@
   "manageAssets": "Manage Assets",
   "manageStudents": "Manage Students",
   "manageStudentsNotificationFailure": "Something went wrong.",
-  "manageStudentsNotificationCannotAdd": "You could not add student to your section. Please try again or refresh the page.",
-  "manageStudentsNotificationAddSuccess": "You added 1 student to your section.",
+  "manageStudentsNotificationCannotAdd": "You could not add {numStudents} students to your section. Please try again or refresh the page.",
+  "manageStudentsNotificationAddSuccess": "You added {numStudents} students to your section.",
   "manageStudentsNotificationSuccess": "Success!",
```

Comment:
tiny nit: Do we want this string to have "student(s)" for cases when numStudents is 1?